StratisCore.UI/src/app/wallet/smart-contracts/components/smart-contracts.component.ts

```diff
@@ -129,7 +129,7 @@ export class SmartContractsComponent implements OnInit {
             .GetReceipt(contract.hash)
             .toPromise()
             .then(result => {
-                this.genericModalService.openModal("Receipt", "<pre>" + JSON.stringify(result, null,"    ") + "</pre>");
+                this.genericModalService.openModal("Receipt", "<pre style='-webkit-user-select: text;'>" + JSON.stringify(result, null,"    ") + "</pre>");
```

Comment:
Tested this today and unfortunately it doesn't seem to work. I think inline styles are stripped out for security reasons, this is the message: `WARNING: sanitizing HTML stripped some content, see http://g.co/ng/security#xss`.

Modified the css with this and it worked, maybe we could add a class to <pre> when we want it to be selectable?
```
    pre {
      white-space: pre-wrap !important;
      -webkit-user-select: text;
      cursor: text;
    }
```